docs/100-upgrade/migration-guides/3.7-3.8.mdx

```diff
@@ -51,3 +51,10 @@ To cope with this update, you should ensure that your codebase contains the
 changes from the fix if your project overrode `/media` routes. See the related
 commit:
 [b445161a](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/commit/b445161ac1880293c9e1e8c6af8a85139c0410b3).
+
+## Update PWA manifest link
+
+In this release, we have fixed an issue with PWA link when the manifest is
+behind a Basic auth, to fix this in your project, you must applu the changes
+made in this commit :
+[35d3bb2f](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/commit/35d3bb2f24679b1fdeb19b936aa488adfd74a630)
```

Comment:
**nitpick:** I'm not a fan of adding an indirection level for small changes

**suggestion:** explicitly provide the diff to apply at project level and keep the link for reference only

Reply:
diff added for better readability